llvm/test/Transforms/InstCombine/AMDGPU/amdgcn-intrinsics.ll

```diff
@@ -1320,6 +1320,30 @@ define <2 x i16> @undef_cvt_pknorm_i16() {
   ret <2 x i16> %cvt
 }
 
+define <2 x i16> @poison_lhs_cvt_pknorm_i16(float %y) {
+; CHECK-LABEL: @poison_lhs_cvt_pknorm_i16(
+; CHECK-NEXT:    ret <2 x i16> poison
+;
+  %cvt = call <2 x i16> @llvm.amdgcn.cvt.pknorm.i16(float poison, float %y)
+  ret <2 x i16> %cvt
```

Comment:
Maybe we should be only propagating poison for the one element. That's how the IR casts would behave

Reply:
But that's poison propagation by element in a single input. This is scattered across arguments. The equivalent generic IR would still be poison